src/PositionService.Business/Commands/Position/EditPositionCommand.cs

```diff
@@ -65,7 +66,7 @@ public async Task<OperationResultResponse<bool>> ExecuteAsync(Guid positionId, J
         return _responseCreator.CreateFailureResponse<bool>(HttpStatusCode.NotFound, errors);
       }
 
-      foreach (var item in request.Operations)
+      foreach (Operation<EditPositionRequest> item in request.Operations)
       {
         if (item.path.EndsWith(nameof(EditPositionRequest.IsActive), StringComparison.OrdinalIgnoreCase) &&
```

Comment:
can you remove it in validator?

Reply:
In the next PR

Reply:
No, because validator does not have positionId, only request.